pkgs/applications/misc/coffeegrindsize/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication {
+  pname = "coffeegrindsize";
+  # no tags in the repo
+  version = "unstable-2021-04-20";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "jgagneastro";
+    repo = "coffeegrindsize";
+    rev = "22661ebd21831dba4cf32bfc6ba59fe3d49f879c";
+    hash = "sha256-HlTw0nmr+VZL6EUX9RJzj253fnAred9LNFNgVHqoAoI=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    tkinter
+    matplotlib
+    numpy
+    pandas
+    pillow
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    echo "#!/usr/bin/env python" > "$out/bin/coffeegrindsize"
+    cat coffeegrindsize.py >> "$out/bin/coffeegrindsize"
+    chmod +x "$out/bin/coffeegrindsize"
+    patchShebangs "$out/bin/coffeegrindsize"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Detects the individual coffee grounds in a white-background picture to determine particle size distribution";
+    mainProgram = "coffeegrindsize";
+    homepage = "https://github.com/jgagneastro/coffeegrindsize";
+    license = licenses.mit;
+    maintainers = with maintainers; [ t4ccer ];
+    platforms = platforms.linux;
```

Comment:
Does tkinter not run on Mac? Or is there some other reason for the platform limitation?

Reply:
I've never used Mac in my life, don't have access to Mac machine, cannot test it

Reply:
That's okay. Only reason I'm asking is because the repository mentions that the application will also run on Macs:

> To open the application simply launch the .app file located at /App/dist/coffeegrindsize.app

You can always set `platforms = platforms.linux ++ platforms.darwin` or `platforms.unix` without testing if there is good reason to believe it will work on Macs or other unix systems. Maybe there is even a reasonable default set by buildPythonApplication (not sure about that).

Reply:
Alright, it defaults to `python.meta.platforms` so probably can be removed